projects/ad411x_ad717x/de10nano/Makefile

```diff
@@ -0,0 +1,23 @@
+#####################################################################################
+## Copyright (c) 2024 Analog Devices, Inc.
```

Comment:
```suggestion
## Copyright (c) 2018 - 2024 Analog Devices, Inc.
```

Reply:
Note about the auto-generated makefile, you can "temporally" revert the lattice commit do use the Makefile generation script (since it was not updated to support the lattice carrier yet)

Reply:
Done. Regenerated Makefile.